index.html

```diff
@@ -2,5 +2,7 @@
   <link rel="stylesheet" type="text/css" href="css/main.css">
   <body>
     <p>Under construction. Please check back again later.</p>
+    <p>For our current site please click the link below</p>
+    <a href="https://uiucneurotech.wixsite.com/website">Visit Our Current Site</a>
```

Comment:
Probably reduce redundant wording "our current site". Could replace line 5 with:

`<p>Click <a href="...">here</a> for our current site.</p>`